internal/core/exchanger/etcd/etcd_test.go

```diff
@@ -22,23 +22,24 @@ import (
 	"testing"
 )
 
-func TestNew(t *testing.T) {
-	config := &config.Config{
-		ExchangerConnMetadata: &config.ConnMetadata{
-			Type:     "etcd",
-			UserName: "root",
-			Password: "root",
-			Host:     "localhost",
-			Port:     "2379",
-		},
+var EtcdConfig = &config.ConnMetadata{
```

Comment:
maybe just file local global var, like 'etcdConnMetadata' instead of `EtcdConfig`